### ISSUE TITLE
ember test fails, even when phantomjs installed globally

### ISSUE THREAD

**Issue by KurtRMueller** [ember-cli/ember-cli#3977]
Hey guys,

I just did a fresh install using `ember-cli`, generated a Ember Data model, and ran my tests via the `ember test` command. I get the following error:

```
1..0
# tests 0
# pass  0
# fail  0

# ok
No tests were run, please check whether any errors occurred in the page (ember test --server) and ensure that you have a test launcher (e.g. PhantomJS) enabled.
```
When I run `ember serve` and go to `http://localhost:4200/tests`, qunit is telling me that 0 tests are being ran. Can anybody help? Thank you!

My Ember data model:

```js
// student.js
import DS from 'ember-data';
export default DS.Model.extend({
  firstName: DS.attr('string'),
  lastName: DS.attr('string'),

  fullName: function() {
    return this.get('firstName') + ' ' + this.get('lastName');
  }.property('firstName', 'lastName')

});
```

```js
// student-test.js

import {
  moduleForModel,
  test
} from 'ember-qunit';

import DS from 'ember-data';
import Ember from 'ember';

moduleForModel('student', 'Student Model', {
  // Specify the other units that are required for this test.
  needs: []
});

test('Student is a valid Ember Data Model', function(assert) {
  var student = this.subject({
    firstName: 'Carlos',
    lastName: 'Danger'
  });
  // var store = this.store();
  assert.ok(student);
  assert.ok(student instanceof DS.Model);
  assert.ok(false);
  assert.equal(true, false);
});
```

Thanks for any help guys.

**Comment by rwjblue** [comment]
Try removing 'assert.ok(student)'.

**Comment by rwjblue** [comment]
In some cases asserting the model itself, causes a bizarre recursive loop in Testem. Asserting like 'assert.ok(!!student)' would work instead (if you still wanted an assertion for this).

**Comment by stefanpenner** [comment]
must be the qunit x testem reported

**Comment by KurtRMueller** [comment]
Removed `assert.ok(student)`. Still 0 tests being ran. I've been reading the docs and they mention something about specifying a `Resolver`. I'm not quite sure what this, but perhaps I'm overlooking something?

**Comment by stefanpenner** [comment]
> I've been reading the docs and they mention something about specifying a Resolver.

not need, tests run out of the box. Sometime else seems funky.

**Comment by KurtRMueller** [comment]
```
╰─➤  ember -v
version: 0.2.3
node: 0.12.2
npm: 2.7.6
```

OS/X Yosemite

I'm about to head home, but I'll whip something together when I get back.

**Comment by KurtRMueller** [comment]
BTW, thanks for all the quick help guys. I know this is hard work.

**Comment by KurtRMueller** [comment]
Okay, weird. I created a new ember cli project, generated a model, ran the tests... and - it works. I have no idea why.

**Comment by stefanpenner** [comment]
> Okay, weird. I created a new ember cli project, generated a model, ran the tests... and - it works. I have no idea why.

in theory, if you can share the app that has issues i can dig in and see whats up

**Comment by Soliah** [comment]
This has started happening to us too, but only for people who've had to recently `npm install` because they're deps were out of date.

Tests work fine with testem and chrome though.

**Comment by foxnewsnetwork** [comment]
I'm also encountering this very problem (except in addon). As requested by @stefanpenner , here's the link to the offending repo:

https://github.com/foxnewsnetwork/ember-dog-food

Note: this might be a bit subversive on my part, that that *is* an addon projects and not a vanilla app project. Nevertheless, it experiences very similar build and test problems (especially using npm test) and complete death on travis-ci. But I should note, in addon, ember serve -> localhost:4200/tests works

**Comment by joeybravo** [comment]
Hi,

Just wanted to mention that I had this problem too. I discovered that since I had cloned/copied my Ember App setup my new app had that old app name in some places/files. I had to find and change ALL instances where my old app name was mentioned, into the correct one for my new app - ex. the index.html still had its HEAD tag links/paths for CSS and JS to the old app (as well as package.json).


So note to self: do not clone ember apps like the wild west still was in effect.

Hope this helps anyone from being as dumb as me.
